gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -91,6 +89,7 @@
   private final Label commandIdx;
   private final SelectionHandler<Control> selectionHandler;
   private final SingleInFlight searchController = new SingleInFlight();
+  private final CommandOptions commandPopup;
```

Comment:
You are not actually using this instance anywhere. So, I suggest:
1) make `CommandOptions` not instantiatable  (i.e. empty private constructor)
2) add a static void function in `CommandOptions` that has all the logic you currently have in the constructor.